web/src/components/atoms/Icon/icons.ts

```diff
@@ -44,4 +54,13 @@ export default {
   plus: PlusOutlined,
   close: CloseOutlined,
   exclamationCircle: ExclamationCircleOutlined,
+  textT: Text,
```

Comment:
textT?

Reply:
Because it has `T` shape, same name Red used in design file.